docs/feature-ideas.md

```
@@ -0,0 +1,385 @@
+
+**Effort:** Medium. Schema migration + backend logic + UI.
+
+---
+
+### 1.5 Gate Progress Bar
+
+**Problem:** No sense of how far through a program the player is.
+
+**Solution:** ASCII progress bar at top of `ProgramPlay`:
+
+```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)